coordinates/sites.json

```diff
@@ -924,5 +924,23 @@
            "John Galt Telescope",
            "DRAO 26m Telescope"
        ]
+   },
+   "Otehiwai Observatory": {
+        "elevation": 1029,
+        "name": "Otehiwai Observatory",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": -47.985447,
+        "elevation_unit": "meter",
+        "longitude": 170.464154,
+        "timezone": "NZ",
+        "source": "Wikipedia https://en.wikipedia.org/wiki/Mount_John_University_Observatory",
```

Comment:
Wikipedia is not necessarily the best source. Anyone can edit the data there any time or links to stuff become dead. For example, the wikipedia site has a link to the observatory website, which does not work. Of course, we can verify the information with a look at google maps (observatory domes are typically pretty easy to spot), but if at all possible, we would strongly prefer a source that people can still look up later, e.g. a publication in the astronomical literature. Of course, sometimes that does not exist, but it's worth checking.

(I know that not all observatories in the already list have a good source, but we try to see what we can do at least for new data added.)

Reply:
Very good point, I've updated it with a reliable source.